api/wd/index.md

```diff
@@ -129,9 +129,9 @@ describe('products', function() {
 });
 ```
 
-#### `browser.loadPage(url, options)`
+### `browser.loadPage(url: string[, options: Object]): Promise<void>`
```

Comment:
```suggestion
### `browser.loadPage(url: string, options?: LoadPageOpts): Promise<void>`
```

Reply:
I'm wondering if we should be more accruate with these types and note that really these return `: Browser & Promise<x>` to show that they're magic chain objects, or if that's needlessly confusing....